file-upload component is missing in v6

I'm getting an error when building with v6. Works fine with previous version.
This is using embroider & webpack & node 14.

Looks like the component file-upload is missing.

  - broccoliBuilderErrorStack: ModuleNotFoundError: Module not found: Error: Can't resolve '../node_modules/ember-file-upload/components/file-upload' in '$TMPDIR/embroider/9ee008/components/file-upload.js'
    at /home/bent/Projekter/FormPlus/node_modules/webpack/lib/Compilation.js:2016:28
    at /home/bent/Projekter/FormPlus/node_modules/webpack/lib/NormalModuleFactory.js:798:13
    at eval (eval at create (/home/bent/Projekter/FormPlus/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /home/bent/Projekter/FormPlus/node_modules/webpack/lib/NormalModuleFactory.js:270:22
    at eval (eval at create (/home/bent/Projekter/FormPlus/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /home/bent/Projekter/FormPlus/node_modules/webpack/lib/NormalModuleFactory.js:434:22
    at /home/bent/Projekter/FormPlus/node_modules/webpack/lib/NormalModuleFactory.js:116:11
    at /home/bent/Projekter/FormPlus/node_modules/webpack/lib/NormalModuleFactory.js:670:25
    at /home/bent/Projekter/FormPlus/node_modules/webpack/lib/NormalModuleFactory.js:855:8
    at /home/bent/Projekter/FormPlus/node_modules/webpack/lib/NormalModuleFactory.js:975:5
  - code: [undefined]
  - codeFrame: Module not found: Error: Can't resolve '../node_modules/ember-file-upload/components/file-upload' in '$TMPDIR/embroider/9ee008/components/file-upload.js'
  - errorMessage: components/file-upload.js: Module not found: Error: Can't resolve '../node_modules/ember-file-upload/components/file-upload' in '$TMPDIR/embroider/9ee008/components/file-upload.js'
        at PackagerRunner (@embroider/webpack)

--- a/ember-file-upload/app/components/file-upload.js
+++ b/ember-file-upload/app/components/file-upload.js
@@ -1,1 +1,0 @@
-export { default } from 'ember-file-upload/components/file-upload';

